packages/base/src/3dview/mainview.tsx

```diff
@@ -1935,40 +1956,71 @@ export class MainView extends React.Component<IProps, IStates> {
             height: 'calc(100%)'
           }}
         />
-        {isTransformOrClipEnabled && (
+
+        {(isTransformOrClipEnabled || this.state.explodedViewEnabled) && (
           <div
             style={{
               position: 'absolute',
               bottom: '10px',
               left: '10px',
+              display: 'flex',
+              flexDirection: 'column',
               padding: '8px',
               backgroundColor: 'rgba(0, 0, 0, 0.5)',
               color: 'white',
               borderRadius: '4px',
-              fontSize: '12px'
+              fontSize: '12px',
+              gap: '8px' // spacing between the two overlays
             }}
           >
-            <div style={{ marginBottom: '2px' }}>Press R to switch mode</div>
-
-            {this.state.transformMode === 'rotate' && (
+            {isTransformOrClipEnabled && (
               <div>
-                <label style={{ marginRight: '8px' }}>Rotation Snap (°):</label>
-                <input
-                  type="number"
-                  value={this.state.rotationSnapValue}
-                  onChange={this._handleSnapChange}
-                  style={{
-                    width: '50px',
-                    padding: '4px',
-                    borderRadius: '4px',
-                    border: '1px solid #ccc',
-                    fontSize: '12px'
-                  }}
-                />
+                <div style={{ marginBottom: '2px' }}>
+                  Press R to switch mode
+                </div>
+                {this.state.transformMode === 'rotate' && (
+                  <div>
+                    <label style={{ marginRight: '8px' }}>
+                      Rotation Snap (°):
+                    </label>
+                    <input
+                      type="number"
+                      value={this.state.rotationSnapValue}
+                      onChange={this._handleSnapChange}
+                      style={{
+                        width: '50px',
+                        padding: '4px',
+                        borderRadius: '4px',
+                        border: '1px solid #ccc',
+                        fontSize: '12px'
+                      }}
+                    />
+                  </div>
+                )}
+              </div>
+            )}
+            {this.state.explodedViewEnabled && (
+              <div>
+                <div style={{ marginBottom: '4px' }}>Exploded view factor:</div>
+                <div style={{ display: 'flex', alignItems: 'center' }}>
+                  <input
+                    type="range"
+                    min="0"
+                    max="10"
```

Comment:
Should we make this 5? On the ArchDetail example 10 seems way too much. Long term, we may want to even let user control the max value.

But the current PR is already a great improvement 👍🏽 let's do that later™

Reply:
Ok I will change it to 5.